### DISCUSSION TITLE
Add discord to player profile if connected

### DISCUSSION THREAD

**Discussion by gnarlyvr1** [gameserverapp/Platform#594] [Suggestions]
Allow admins to see a Player's discord account name under player information as well as add {discordID} as a variable. It can be a great  way to get in contact and find players if needed as well as provide additional functionality for integration with other/custom discord bots.

**Comment by mvaessen** [comment]
Hi, any connected social account is listed on the player page.
Task actions for discord come with a `{user}` tag which lets you mention the player.

![Screenshot_2023-11-27_at_20_33_29](https://github.com/gameserverapp/Platform/assets/13402423/fa77e9aa-c15a-49e2-8f42-76382e4bff1e)

**Reply by gnarlyvr1** [reply]
I'm an idiot, disregard. Thanks!
